Differential Gene Expression: Upregulation and Downregulation

Biomarkers 

Reem Atawia

Despite the wide use of Polymerase Chain Reaction in determining the effect of certain conditions on single gene expression, the current use of DNA microarray and RNA sequencing is increasing rapidly. Microarrays and sequencing help identify the changes in a set of genes, which is known as differential gene expression. If a group of genes is differentially expressed in certain conditions, this indicates that the observed normalized read count or expression level is statistically significant between two or more sets of experimental conditions. The tools used for differential expression include edgeR, DESeq, and maSigPro. Heatmaps and clustering are commonly used to visualize gene expression, and data are usually displayed as a grid. The change in gene expression, known as upregulation, or the decrease in gene expression, known as downregulation, is displayed by different box colors, and the color intensity refers to the degree of gene expression. To further interpret the results of gene expression, gene set enrichment and pathway analyses are used. The enrichment analysis of differentially expressed genes is performed using Gene Ontology to identify the enrichment in biological processes, molecular functions, and cellular components. The common tools used for enrichment and pathway analysis include DAVID and Reactome<sup>1</sup>.

Examples of applications of differential gene expression include the identification of dysregulated pathways or a molecular mechanism in the pathogenesis of certain diseases, which will eventually help in the development of novel biomarkers useful in the diagnosis or the prognosis of the disease. Additionally, differential gene expression can help in the identification of novel therapeutic options to target certain pathways that are differentially expressed in the disease. Determination of differentially expressed genes can help identify gene clusters regulating embryonic development or tissue differentiation and, thus, help better understand these processes. These analyses are essential for understanding of various biological processes, disease mechanisms, and the advancement of precision medicine.

Despite the great advantage of differential gene expression to the scientific community, scientists encounter some challenges, including data reproducibility and the requirement for robust statistical methodology. Additionally, the bioinformatics analysis of the data generated from RNA-seq is time-consuming, especially for biologists without a computational background. However, some user-friendly web applications are currently available to facilitate the identification and visualization of the differentially expressed genes, such as Degust and iDEP<sup>2</sup>. Collectively, despite the current limitation, differential gene expression continues to play as essential role in advancing the understanding of disease pathophysiology and drug development.

**_References_**

1-Chung, Matthew, et al. "Best practices on the differential expression analysis of multi-species RNA-seq." _Genome biology_ 22.1 (2021): 1-23.

2-Steven Xijin Ge, etal. "iDEP: an integrated web application for differential expression and pathway analysis of RNA-Seq data. " BMC bioinformatics (2018)

 
